app/bundles/PageBundle/Tests/Controller/PublicControllerTest.php

```diff
@@ -266,8 +265,10 @@ private function getVariantContent($aCount, $bCount, $cCount)
             );
 
         $container = $this->createMock(Container::class);
-        $container->method('has')
-            ->will($this->returnValue(true));
+        $container->expects(self::never())
+            ->method('has'); // THIS CAN FAIL!!!!!!!!!!!!!!
```

Comment:
What shall we do with this? Do you suggest to merge this or is this just a type of "todo" comment?

Reply:
Yes, sorry, that was a comment for me. Removed.